### DISCUSSION TITLE
"Dashboard" view for app - Time and Date at the top!

### DISCUSSION THREAD

**Discussion by andiohn** [breezy-weather/breezy-weather#1183] [Ideas]
### Describe your idea

I would love to use this app as a dashboard directly on e-ink display. All I would need is time at the top and maybe date right below it, with selectable fonts in the future! The card system already looks nice for this purpose, and what if we had an auto scroll every two minutes to show another card while keeping the time at the top. 

If we could make it fully immersive, and then just have the time, and then disable the background, that would do it! A future improvement would be to also have a calendar card, where you could show Google calendar or something like that, but that sounds like pretty big feature to add. 

Short-term Goal:
1. Time At the top of the main display with check box to display or not. Perhaps it's just another card! Turn on and off date.
2. Disable background toggle, which would then show white or black?

Medium-term Goal:
1. "Sticky" setting for clock card, to keep it at the top always
2. Font choices, or Custom font perhaps

Long-term Goal:
1. "Auto scroll" to cycle cards every X minutes. Keeping top card if stickiness toggled.
2. Calendar events card for today events, etc. Local calendar or Google calendar or whatever.
3. "Home Assistant buttons" card...

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.5 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by andiohn** [comment]
Simply put, the first quick win would be a clock card! Thanks for an AWESOME open sauce app :)

**Comment by papjul** [comment]
It's very specific, it's like designing a new app.

Fortunately, a good thing is that we expose weather data through a broadcast, so you can write an e-ink-friendly app that registers a receiver of the Gadgetbridge WeatherSpec and just focus on the design while we focus on handling all the logic ;)

**Reply by andiohn** [reply]
What about just adding a card with the time and date? I wonder if that shouldn't be to bad? I might look into it myself too, I'm not a freeloader :)

**Reply by papjul** [reply]
Duplicate of #304

**Comment by papjul** [comment]
A basic clock block was added in v6.0.3-alpha.
Some design improvements are planned as part of #1984
